app/services/irs_attempts_api/attempt_event.rb

```diff
@@ -24,6 +24,7 @@ def to_jwe
         event_data_encryption_key,
         typ: 'secevent+jwe',
         zip: 'DEF',
+        enc: 'A256GCM',
```

Comment:
the plan is still to asymmetrically encrypt individual events right? this would symmetrically encrypt them?

Reply:
The default here is `A128CBC-HS256`, so this should just be a switch to be closer to our other symmetric encryption strategies.  My understanding is both prior and with this change, the Content Encryption Key is asymmetrically encrypted with the public key, this just changes that the CEK uses a different encryption algorithm to symmetrically encrypt the message.